### PR TITLE
docs: address PE-4011 and PE-3874

### PR DESCRIPTION
**Pull request by lennessyy**
## Describe the Change

This PR documents a known issue for PE-4011. Updates to the OS pack and K8s pack in the stages section trigger a repave instead of a reboot. 

## Changed Pages

💻 [Add Preview URL for Page]()

## Jira Tickets

🎫 [PE-4011](https://spectrocloud.atlassian.net/browse/PE-4011)
🎫 [PE-3874](https://spectrocloud.atlassian.net/browse/PE-3874)

## Backports

Can this PR be backported?

- [ ] No. This only applies to 4.3. 

[PE-4011]: https://spectrocloud.atlassian.net/browse/PE-4011?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[PE-3874]: https://spectrocloud.atlassian.net/browse/PE-3874?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/clusters/edge/upgrade-behavior.md [hunk 1]
```diff
@@ -12,6 +12,20 @@ profile. Depending on the nature of the change, implementing an upgrade might in
 cluster, restarting services, or doing nothing. For more information about cluster repaves, refer to
 [Repave Behavior and Configurations](../cluster-management/node-pool.md#repave-behavior-and-configuration).
 
+:::warning
+
+- If a cluster's Kubernetes service is down, updates to the cluster's profile will not get applied. You must fix the
+  issue impacting the Kubernetes service first and ensure that the Kubernetes service of the cluster is operational
+  before applying updates to the cluster.
```
Comment (yuliiiah) [inline docs/docs-content/clusters/edge/upgrade-behavior.md:19]: Should this be a list item? 👀

Comment (yuliiiah) [reply docs/docs-content/clusters/edge/upgrade-behavior.md:19]: I'm a bit confused with the note: If the k8s service is down, Palette will not apply updates from the cluster profile to the cluster? Or updates to the cluster profile itself? Then, once you fix the k8s service, do you have to manually apply the cluster profile updates? I think this is where I'm lost:

- Who/what doesn't apply the updates if the k8s service is down?
- What is being updated, the deployed cluster or the cluster profile?
- What updates should the used apply once the k8s service is up again?

**Review by yuliiiah — APPROVED** [review]
Approving with some questions to the warning ✨

File: docs/docs-content/clusters/edge/upgrade-behavior.md (hunk 1 above)
Comment (lennessyy) [reply docs/docs-content/clusters/edge/upgrade-behavior.md:19]: It applies to updates to the cluster. When you update a profile, the updates in the profile get applied to the cluster. If K8s is down, the updates won't be applied to the cluster.

**Pull request merged**